### PR TITLE
Phase 3/5: 4-value <position> edge-offsets + background-origin + background-clip + PR #170 review

### PR DESCRIPTION
**Pull request by raroche**
Three background/position deferral riders, one PR (off `main` @ 7ed7807).

## (A) 4-value edge-offset `<position>` forms (edge-offset cycle, CSS B&B §3.6)
The shared `FragmentPainter.TryParseBackgroundPosition` (used by **both** `object-position` and `background-position`) now parses the 3-/4-value edge-offset form — an edge keyword each optionally followed by a `<length-percentage>` offset FROM that edge (`left 10px top 5px`, `right 25% bottom 50%`). Components are assigned to axes by keyword (left/right → X, top/bottom → Y, center → the free axis), in either order; two same-axis edges or a leftover token reject.

**Reachability:** facade-reachable via **margin-box** backgrounds (raw values bypass AngleSharp). A BODY `background-position` or an `object-position` 4-value form is AngleSharp-dropped at parse — a documented beta gremlin; only the parser + the margin-box raw path see it.

## (B) background-origin (bg-origin cycle, CSS B&B §3.4)
A body block's `background-image` tiles over the **positioning area** set by `background-origin` — `border-box` / `padding-box` (the initial) / `content-box`, the border box inset by the **used** border (padding-box) or border+padding (content-box). New `BackgroundAreaInset` + `UsedBorderEdgeWidthPx` (the inset matches what the border painter actually paints — a `none`-style edge contributes 0). Replaces the old border-box approximation; **borderless boxes are byte-identical**.

## (C) background-clip (bg-clip cycle, CSS B&B §3.5)
The **paint area** (the tile clip + the pattern-fill bounds) is the `background-clip` box — `border-box` (the initial) / `padding-box` / `content-box` — via new optional clip-rect params on the tiler.

## Tests
- **Unit:** +11 `BackgroundVariantParserTests` Position rows (7 edge-offset supported + 4 reject incl. two-same-axis / 5-token).
- **Facade:** margin-box `background-position: left 10px top 5px` (+7.5pt X / −3.75pt Y); `background-origin` shift (border→padding +3pt, →content +9pt); `background-clip` rect (border 78pt / padding 72pt / content 60pt); the `object-position` unsupported-form test switched to a genuinely-invalid two-X-edge value.

## Gates
6680 unit / 6 skip · 30 LayoutSnapshots · 97 RealDocuments · AOT/JIT parity 4 (byte-identical, no re-pin) all green; Release **0 warnings**.

Docs: `PROGRESS.md`, `CLAUDE.md`, `docs/deferrals.md`, `docs/compatibility-matrix.md`. Deferred: margin-box `background-origin`/`-clip` (the band stays the area); the BODY + `object-position` 4-value position forms (AngleSharp drop).

🤖 Generated with [Claude Code](https://claude.com/claude-code)

### CODE REVIEW

File: src/NetPdf/Rendering/FragmentPainter.cs [hunk 1]
```diff
@@ -147,19 +156,31 @@ public static void PaintFragments(
...
-        string? repeatRaw = null, string? sizeRaw = null, string? positionRaw = null)
+        string? repeatRaw = null, string? sizeRaw = null, string? positionRaw = null,
+        double? clipLeftPx = null, double? clipTopPx = null,
+        double? clipWidthPx = null, double? clipHeightPx = null)
     {
+        // background-clip (bg-clip cycle): the PAINT area, defaulting to the POSITIONING area —
+        // the body caller passes the border box here while the area (leftPx..heightPx) is the
+        // background-origin box; the margin-box caller leaves it = the band. Tiles are POSITIONED
+        // in the area (background-origin) but PAINTED only within the clip rect.
+        var clipL = clipLeftPx ?? leftPx;
```
[the remark this answers is not in the document]
Comment (raroche) [reply src/NetPdf/Rendering/FragmentPainter.cs:180]: Fixed in 56347a3. `AxisTilingPlan` gained a coverage window `[coverStartPx, coverEndPx]` (the background-clip rect, area-relative): `repeat` now tiles to cover the clip, the grid still PHASED at posPx in the positioning area; `space`/`round` still fill the positioning area per §3.6.1. Added loop-path + pattern-path facade tests + an AxisTilingPlan repeat-coverage unit test.

File: src/NetPdf/Rendering/FragmentPainter.cs [hunk 2]
```diff
@@ -221,6 +242,11 @@ internal static void PaintBackgroundImageTiles(
             var fillRightPx = singleX ? Math.Min(leftPx + widthPx, leftPx + firstX + tileW) : leftPx + widthPx;
             var fillTopPx = singleY ? Math.Max(topPx, topPx + firstY) : topPx;
             var fillBottomPx = singleY ? Math.Min(topPx + heightPx, topPx + firstY + tileH) : topPx + heightPx;
+            // Bound the pattern fill by the background-clip rect (bg-clip cycle).
```
[the remark this answers is not in the document]
Comment (raroche) [reply src/NetPdf/Rendering/FragmentPainter.cs]: Fixed in 56347a3. The pattern fill bounds now come from the TILED extent (`leftPx + firstX` → the last tile's far edge) intersected with the clip rect — a repeating axis spans the clip, a non-repeating axis the single tile ∩ clip. The pattern-path facade test asserts the 78pt border-box fill.
